src/arrayMethodSort.js

```diff
@@ -4,9 +4,61 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compare =
+      compareFunction !== undefined
+        ? compareFunction
+        : (item1, item2) => (String(item1) > String(item2) ? 1 : -1);
```

Comment:
Ensure that the `compareFunction` is correctly handled. The current implementation seems correct, but make sure it aligns with your requirements for sorting when `compareFunction` is undefined.